corehq/apps/reports/static/reports/v2/js/columns.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * todo add docstring
+ */
+
+hqDefine('reports/v2/js/columns', [
+  'lodash',  // can we switch to lodash?
+  'reports/v2/js/formatters',
+], function (
+    _,
+    Formatters
+) {
+  'use strict';
+
+  var Column = function (data) {
```

Comment:
Another new convention: lowercase first letter for function-based classes.

Reply:
fixed